EDI/UserInterface/03a_LISTLogging.md

```diff
@@ -0,0 +1,163 @@
+# LIST (Log)
+
+*DataHub Exporter > Log > Tab LIST*
+
+![Log](../../Assets/Screenshots/EDI/UserInterface/Log.png "[Log]")
+
+**List of intents**   
+The *List of intents* contains all intents. An intent is a summary of log entries generated by one system action. Depending on the settings, the displayed columns may vary. All fields are read-only.
+
+- *Name*  
+    Description of the intent.
+
+- *Number of messages*  
+    Number of messages contained in the intent.
+
+- *Created at*  
+    Date and time of the creation.
+
+- *Created by*  
+    Name and username of the user that initiated the action leading to the intent. If the intent was created automatically by the system, the field is empty. 
+
+- *Max log level of message*  
+    Maximum log level type of the message(s) contained in the intent. The following levels are available:   
+    - **Debug**   
+    - **Info**
+    - **Notice**
+    - **Warning**
+    - **Error**
+    - **Critical**
+    - **Alert**
+    - **Emergency**  
+        
+- *ID*  
+    Intent identification number. The ID number is automatically assigned by the system.
+
+- *Short preview*  
+    First 30 characters of the log message.
+
+- *Preview*  
+    Complete log message.
+
+
+## List of messages
+
+*Payments > Logging > Tab LIST > Select an intent*
+
+![List of messages](../../Assets/Screenshots/EDI/UserInterface/LogListOfMessages.png "[List of messages]")
+
+**List of messages**   
+
```

Comment:
Keine Buttons in editing toolbar? [DOWNLOAD] [VIEW]?

Reply:
done